### DISCUSSION TITLE
where do i explicitly import `NuxtLinkLocale` from (without auto-imports)

### DISCUSSION THREAD

**Discussion by stefanprobst** [nuxt-modules/i18n#3190] [Q&A]
where do i explicitly import `NuxtLinkLocale` from (without auto-imports)?

it's not documented on https://i18n.nuxtjs.org/docs/api/components#nuxtlinklocale

**Comment by rendisimamora** [comment]
try this

`import` { NuxtLinkLocale } from 'nuxt-i18n/components'

```
export default {
  components: {
    NuxtLinkLocale
  },
  // other component options...
}

```

**Reply by stefanprobst** [reply]
this obviously would not work - the package is not called `nuxt-i18n` but `@nuxtjs/i18n` (which does not provide a `components` export).

![nuxtlinklocale](https://github.com/user-attachments/assets/c367aedc-8121-4542-b9fc-6a8059770d3e)
